### ISSUE TITLE
Option to not use Asset table

### ISSUE THREAD

**Issue by fun4uskids** [vdm-io/Joomla-Component-Builder#314]
### Steps to reproduce the issue
I would like an option to not require my component tables to be associated with the joomla assets table.

### System information (as much as possible)
- OS Name & Version: 
- MySql Version: 
- Apache Version: 
- PHP Version: 
- Joomla Version: 
- JCB Version: 
- Browser:

**Comment by Llewellynvdm** [comment]
Okay just that we are sure you know what you are asking for.

If the items are not linked to the asset table these are all the features you will lose:

- No history integration.
- Broken permissions control.
- Limited Joomla integration.

Did you know that if you uninstall a component build by JCB it also removes these asset relationships.
If all you want is a database to store data, unrelated to any really Joomla integration, why not use a table dump like we did in JCB:
![image](https://user-images.githubusercontent.com/5607939/42390303-59da10f8-8143-11e8-8718-6e7ac46f10ed.png)

So if you understand the implications of this removal of asset relationship then we will start looking at implementation and possible breakages.

**Comment by fun4uskids** [comment]
There are lots of joomla components that do not use the asset table and
they work just fine.

My problem is that I have 20+ sites I am trying to keep synchronized.  If
anything is built/add/updated in the wrong order then the asset tables end
up with different IDs and then I have synchronization issues.

Thanks,
J Wadsworth
352-283-0615

**Comment by Llewellynvdm** [comment]
I did not say it can't work, just that you lose features that depend on the asset table, and since JCB was build to add those features, taking the asset value out of the database, and blocking relationships from being made, we will probably run into breakages.

Interesting I have been working on a project called sync portal that keeps Joomla websites in sync with each other. This is a JCB component, wonder if you would be interested to help finish it. It can already set hash values of a million records in under 2 minutes. I abandoned it do to other responsibilities at the time, but I know if I could finish this it will be a huge hit.

I have already build a similar technology with the JCB export and import area. I don't know if you realize but I am remapping all ids and their relationships upon import of any JCB package. Do your self a favour and go [review the code](https://github.com/vdm-io/Joomla-Component-Builder/blob/staging/admin/models/import_joomla_components.php). Believe me, this area of JCB is mind blowing.
